app/dev-portal/api-catalog-and-packaging.md

```diff
@@ -0,0 +1,179 @@
+
+### API package limitations
+
+When configuring API packages, keep the following limitations in mind:
+
+* If you have two endpoints in your spec, one with a trailing slash one without, both are counted as the same operation.
+* If two operations have the same path and method, they are marked as the same operation.
+* All operations for a control plane are removed when you unlink the control plane.
+* Control planes can only be mapped to one API at a time.
+* You can't change a version or edit a spec for an API if it would remove an operation. The recommended method to remove an operation is to unlink the control plane from the API.
+
+## Package APIs with Dev Portal
+
+This guide shows you how to create an API package in the {{site.konnect_short_name}} UI.
+
+### Prerequisites
+
+To package APIs with Dev Portal, you need:
+* A v3 Dev Portal created in {{site.konnect_short_name}}
+* A [Gateway Service](/gateway/entities/service/#set-up-a-gateway-service) and [Route](/gateway/entities/route/#set-up-a-route) added to your control plane
+* The following roles:
+  * Editor role for APIs
+  * Publisher role for the API and API package
+* [An API created](/dev-portal/apis/) with an [OpenAPI spec](/dev-portal/apis/#api-specs) associated with it
+
+### Associate a control plane
+
+1. In {{site.konnect_short_name}}, navigate to **Dev Portal** > **APIs** in the sidebar.
+1. Click your API.
+1. Click the **Gateway** tab.
+1. Click **Link gateway**.
+1. Select a control plane from the **Control plane** dropdown menu.
+1. Select **Link to a control plane**.
+1. In the Add the Access Control and Enforcement plugin settings, click **Add plugin**.
+1. Click **Link gateway**.
+
+Your operations should now be autogenerated based on how your OpenAPI spec maps to your Routes.
+
+### Assign operations to API packages
+
+1. In {{site.konnect_short_name}}, navigate to **Dev Portal** > **APIs** in the sidebar.
+1. Click the **API packages** tab.
+1. Click **Create API package**.
+1. Enter a name in the **API package name** field.
+1. Enable the Package rate limit and configure your rate limit.
+1. Click **Add operations from APIs** in the API operations settings.
+1. In the Add API operations pane, click your API and click **Add** next to the operations you want to package.
+1. Click **Create API package**. 
+
+### Publish packages to Dev Portal
+
+1. In {{site.konnect_short_name}}, navigate to **Dev Portal** > **APIs** in the sidebar.
+1. Click the **API packages** tab.
+1. Click your API package.
+1. Click **Publish API**.
+1. Select your Dev Portal from the **Portal** dropdown menu.
+1. Select an auth strategy from the **Authentication strategy** dropdown menu.
+1. Click **Publish API**. 
+
+Your API package will now be published to your Dev Portal.
```

Comment:
Could add context that API packages published to the portal behave the same as APIs published to the portal in terms of their display in the API catalog and ability to be registered for by applications. Understood if this is seen as implied.